server/middlewares/authentication.js

```diff
@@ -42,8 +42,8 @@ export const adminAuth = (req, res, next) => {
 };
 
 export const staffAuth = (req, res, next) => {
-  const { type } = req.user;
-  if (type !== 'staff') {
+  const { type, isadmin } = req.user;
+  if (type !== 'staff' || isadmin === true) {
```

Comment:
Does this mean **`admins`** cannot perform transactions?

Reply:
Yes. Only `staff(cashiers)` are allowed to perform transactions

Reply:
Okay... Sounds good